style.css

```diff
@@ -0,0 +1,154 @@
+* {
+    box-sizing: border-box;
+    margin: auto;
+    padding: 0;
+  }
+
+html {
+    background-color: rgb(226, 226, 226); 
+}
+
+body {
+    background-color:  rgba(230, 214, 255, 0.288);
+    max-width: 65%;
+    }
+
+.header-image {
+    border-radius: 50%;
+    width: 10%;
+    height: 10%;
+    float: left;
+    padding: 10px
+}
+
+.social-media-links {
+    float: right;
+    border-radius: 50%;
+    padding-right: 10px;
+}
+
+.header-image2 {
+    width: 90%;
+    padding: 0 20px 0 20px;
+    height: 209px;
+    display: block;
+}
+
+.header, h1 {
+    /* float:left; */
+    padding-top: 10px;
+}
+
+.clearfix {
+    clear:both;
+}
+
+.menu {
+    padding-top: 5px;
+}
+
+header {
+    padding-top: 20px;
+}
+
+.body h2 {
+    font-size: 2rem;
+    padding-left: 5px;
+}
+
+.body h3 {
+    padding-left: 5px;
+}
+
+h4 {
+    padding-bottom: 17px;
+    padding-left: 5px;
+}
+
+.subheader {
+    margin-left: 2%;
+    height: 185px;
+    border: 7px solid black;
+    float: left;
+    border-radius: 50%;
+    padding: 35px;
+    margin-bottom: 3%;
```

Comment:
Using percentages for vertical margins/padding is not a great idea -- the percentage is based off the current height of the browser, which probably isn't what you want.